music/src/core/data.ts

```diff
@@ -190,7 +190,8 @@ export class DrumsConverter extends DataConverter {
 
   constructor(args: DrumsConverterArgs) {
     super(args);
-    this.pitchClasses = args.pitchClasses === undefined  ?
+    this.pitchClasses =
+        (args.pitchClasses === null || args.pitchClasses === undefined) ?
```

Comment:
Does just `!args.pitchClasses ? ` work here?

Reply:
that's too easy